test/lib/services/html_replace_marker_test.rb

```diff
@@ -73,19 +73,17 @@ def test_revert_input_empty_value
 
     def test_revert_multiple_input
       marker = HtmlReplaceMarker.new
-      original_html = [
-        '<html><body>',
-        '<input type="hidden" value="please_revert1"></body></html>',
-        '<input type="hidden" value="please_revert2"></body></html>',
-        '<input type="hidden" value=""></body></html>'
-      ].join
-      new_html = [
-        '<html><body>',
-        "<input type=\"hidden\" value=\"#{marker.add_value('please_revert1')}\"></body></html>",
-        "<input type=\"hidden\" value=\"#{marker.add_value('please_revert2')}\"></body></html>",
-        "<input type=\"hidden\" value=\"#{marker.add_value('')}\">",
-        '</body></html>'
-      ].join
+      original_html = '<html><body>'
+      new_html = '<html><body>'
+
+      50.times do |i|
+        original_html += "<input type=\"hidden\" value=\"#{i}\"></body></html>"
+        new_html += "<input type=\"hidden\" value=\"#{marker.add_value(i.to_s)}\"></body></html>"
+      end
+
+      original_html += '</body></html>'
+      new_html += '</body></html>'
+
```

Comment:
Revert is called in `wovnrb.rb` but the API response is mocked to return the reverted HTML:
https://github.com/WOVNio/wovnrb/blob/b5fe58caa723bfe36a920e349dde5b0e7df4dffa/test/lib/wovnrb_test.rb#L302-L308

I think it makes sense to adjust this test since the revert logic is here anyway.